### ISSUE TITLE
iOS works ok, but Android app issues "Connect Timeout has Expired" message on startup and nothing loads

### ISSUE THREAD

**Issue by Lee-Hounshell-at-MarianaTek** [Kotlin/kmp-production-sample#55]
Increasing the connectTimeout at line 13 of AndroidHttpClient.kt does not fix it.

I see this in logcat:
2022-09-02 09:57:44.604 16529-16529/com.github.jetbrains.rssreader.androidApp V/AndroidHttpClient: REQUEST https://blog.jetbrains.com/kotlin/feed/ failed with exception: io.ktor.client.network.sockets.ConnectTimeoutException: Connect timeout has expired [url=https://blog.jetbrains.com/kotlin/feed/, connect_timeout=unknown ms]
2022-09-02 09:57:44.608 16529-16529/com.github.jetbrains.rssreader.androidApp D/FeedStore: Action: Error(error=io.ktor.client.network.sockets.ConnectTimeoutException: Connect timeout has expired [url=https://blog.jetbrains.com/kotlin/feed/, connect_timeout=unknown ms])

**Comment by PROGERPEN** [comment]
Yep, that's true. The problem is in line isMinifyEnabled = true. I think that's because entity change their field, but I'm not sure. You can change isMinifyEnabled to false and it'll be working. 
```
getByName("release") {
            isMinifyEnabled = true
            signingConfig = signingConfigs.getByName("release")

            proguardFiles(
                getDefaultProguardFile("proguard-android-optimize.txt"),
                file("proguard-rules.pro")
            )
        }
```

**Comment by pahill** [comment]
This is fixed, I checked with debugPG and minifiedEnabled=true. Quite possibly the original bug was fixed.
